bin/smack.py

```diff
@@ -141,6 +144,9 @@ def arguments():
   verifier_group.add_argument('--loop-limit', metavar='N', default='1', type=int,
     help='upper bound on minimum loop iterations [default: %(default)s]')
 
+  verifier_group.add_argument('--context-switches', metavar='k', default='1', type=int,
```

Comment:
This flag is improperly named. Corral’s `/k:int` flag specifies the number of contexts to allow during a single execution. The number of “context switches” is one less than the number of contexts. For example, in an execution with one context, there are zero context switches.